recipes/clingo/meta.yaml

```diff
@@ -8,7 +8,6 @@ package:
 
 source:
   url: https://github.com/potassco/clingo/archive/v{{ version }}.tar.gz
-  sha256: b0d406d2809352caef7fccf69e8864d55e81ee84f4888b0744894977f703f976
```

Comment:
you need to have a sha!

Reply:
I know. I upgraded the version and was too lazy to calculate it for the new archive right away. Anyway, I have to make it work again first. On linux and macos it compiles but then conda complains about library issues. On windows it does not even start to build. I have to compare with other recipes and figure out what is going wrong. Adding back the sha256 will be the least problem...